conf/machine/include/rpi-default-providers.inc

```diff
@@ -1,6 +1,7 @@
 # RaspberryPi BSP default providers
 
 PREFERRED_PROVIDER_virtual/kernel ?= "linux-raspberrypi"
+PREFERRED_PROVIDER_virtual/kernel_poky-tiny ?= "linux-raspberrypi"
```

Comment:
This is a distro specific override, I wonder why is this needed, The right fix it to change it in meta-poky from = to ?=
see
https://git.yoctoproject.org/cgit/cgit.cgi/meta-yocto/tree/meta-poky/conf/distro/poky-tiny.conf#n40

Reply:
well i know that. Also you can set the provider in local.conf. However other layers such as meta-intel do the same in their machine config, providing out-of-the-box experience when compiling poky-tiny.

Reply:
If this was fixed in distribution layer as suggested then bsp and other layers don’t have to do it meta-intel doing it doesn’t give legitimacy to do it elsewhere

Reply:
Yeah I'm convinced. So I will try to get it upstream in poky.